pjmedia/src/pjmedia/transport_loop.c

```diff
@@ -239,7 +240,6 @@ static void tp_loop_on_destroy(void *arg)
     struct transport_loop *loop = (struct transport_loop*) arg;
 
     PJ_LOG(4, (loop->base.name, "Loop transport destroyed"));
-    pj_pool_release(loop->pool);
```

Comment:
No need to erase this, it will cause leak.

Reply:
But then it is released twice: the second time in https://github.com/pjsip/pjproject/blob/ca5255795a330f88d72a0504d902de8bbddcdfeb/pjlib/src/pj/lock.c#L404.

Reply:
the issue is that, the pool is released when the tp is still in use. If the tp is not used any longer, then it will be safe to release the pool. Tested here without removing the pool release, and there's no crash

Reply:
But then we have two releases of pool - or do I miss something?
`pj_grp_lock_dec_ref()` -> `grp_lock_dec_ref()`-> `grp_lock_destroy()`:
*  `cb->handler()`-> `tp_loop_on_destroy()` -> `pj_pool_release()`.
*  `pj_pool_release()`.


I have continue debugging...

Anyway... due to https://github.com/pjsip/pjproject/commit/6dc9b8c181aff39845f02b4626e0812820d4ef0d I get a crash for a code that worked at least since 2.11.

Reply:
You are right  - it is working for me too.